status/controls.py

```diff
@@ -0,0 +1,113 @@
+"""
+    Handler related to Controls page
+"""
+from status.util import SafeHandler
+from genologics import lims
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+lims = lims.Lims(BASEURI, USERNAME, PASSWORD)
+
+class ControlsHandler(SafeHandler):
+
+    def get(self):
+        t = self.application.loader.load('controls.html')
+
+        # get information from databases
+        ws_data, ws_name_data = self.worksets_data()
+        neg_control_data = self.find_control_data('negative control')
+        pos_control_data = self.find_control_data('positive control')
+
+        # create a dictionary with all control data in the correct format
+        all_control_data = {}
+        all_control_data["negative"] = self.add_workset_project(neg_control_data, ws_data, ws_name_data)
+        all_control_data["positive"] = self.add_workset_project(pos_control_data, ws_data, ws_name_data)
```

Comment:
Couldn't `find_control_data` be called inside `add_workset_project` ? Then this could be 
```suggestion
        # create a dictionary with all control data in the correct format
        all_control_data = {}
        for option in ["negative", "positive"]:
            all_control_data[option] = self.add_workset_project(option, ws_data, ws_name_data)
```